src/test/ui/error-codes/E0223.stderr

```diff
@@ -2,9 +2,10 @@ error[E0223]: ambiguous associated type
   --> $DIR/E0223.rs:14:14
    |
 LL |     let foo: MyTrait::X;
-   |              ^^^^^^^^^^ ambiguous associated type
-   |
-   = note: specify the type using the syntax `<Type as MyTrait>::X`
+   |              ^^^^^^^^^^
+   |              |
+   |              ambiguous associated type
+   |              help: use fully-qualified syntax: `<Type as MyTrait>::X`
```

Comment:
Is `Type` a default? If it is, then it shouldn't be machine applicable in those cases.

Reply:
[Good](https://github.com/rust-lang/rust/blob/77af314083e5acabf9ba5335e47271f35eef2e99/src/librustc_typeck/astconv.rs#L1288) [catch](https://github.com/rust-lang/rust/blob/77af314083e5acabf9ba5335e47271f35eef2e99/src/librustc_typeck/astconv.rs#L1242) :baseball: :broken_heart: